.github/workflows/main.yml

```diff
@@ -48,6 +48,18 @@ jobs:
       NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
 
+  create-update-issues:
+    name: Create update issues
+    needs: [is-release, publish-release]
+    if: needs.is-release.outputs.IS_RELEASE == 'true'
+    permissions:
+      issues: write
+    uses: ./.github/workflows/create-update-issues.yaml
+    with:
```

Comment:
Before merge security team have to create new github app, I will send request to @NicholasEllul